_includes/help/nav_sidenav.html

```diff
@@ -5,14 +5,13 @@
       {% for subpage in site.help_pages %}
       {% assign subpage_slug = subpage | slugify %}
       {% assign related_questions = site.help | where_exp: "item", "item.url contains subpage_slug" | sort: 'order' %}
-      {% capture is_initially_expanded %}
-      {% if page.url contains subpage_slug %}true{% else %}false
+      {% if page.url contains subpage_slug %}
+        {% assign is_initially_expanded = 'true' %}
+        {% assign is_initially_hidden = 'false' %}
+      {% else %}
+        {% assign is_initially_expanded = 'false' %}
+        {% assign is_initially_hidden = 'true' %}
       {% endif %}
-      {% endcapture %}
-      {% capture is_initially_hidden %}
-      {% if page.url contains subpage_slug %}false{% else %}true
-      {% endif %}
-      {% endcapture %}
       <li class="usa-sidenav__item usa-parent {% if page.url contains subpage_slug %}usa-current{% else %} {% endif %}"
         aria-expanded="{{ is_initially_expanded }}">
         {{ site["translations"][site.lang]["help_subpages"][subpage] }}
```

Comment:
I can't get it in the diff...but below we have:

```html
      <div aria-hidden={{ is_initially_hidden }}>
```

Can we quote that while we're here? ⬇️ 

```html
      <div aria-hidden="{{ is_initially_hidden }}">
```

Reply:
If you can get this working with quotes that would be awesome! I initially had it in quotes and it wouldn't render. Any idea why?

Reply:
> If you can get this working with quotes that would be awesome! I initially had it in quotes and it wouldn't render. Any idea why?

Interesting, not sure why that'd be an issue. If anything, I'd expect conflicts with the quotes omitted.

In any case, it appears to work well in my testing with the quotes, so I'll add them in.